persistence-typed-tests/src/test/scala/org/apache/pekko/persistence/typed/scaladsl/EventSourcedBehaviorSignalSpec.scala

```diff
@@ -0,0 +1,205 @@
+/*
```

Comment:
Is this test case based on a copy/paste of existing class? If not then https://github.com/apache/pekko/blob/main/project/AddMetaInfLicenseFiles.scala#L1-L16 is the right source header.
If it based on an existing source file then this header is still possibly wrong as it omits the Lightbend copyright source header.

Reply:
Mostly original, changed licence header